openlogi.gemspec

```diff
@@ -28,6 +28,9 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "typhoeus", "~> 1.0.1"
+  spec.add_dependency "activesupport"
+  spec.add_dependency "json"
+  spec.add_dependency "virtus"
```

Comment:
Sorry, for tiny point.

That will be display "WARNING" when do `gem build openlogi`.
We should decide gem versions :wink: